Dockerfile

```diff
@@ -6,14 +6,14 @@ ENV LC_CTYPE C.UTF-8
 
 FROM base as builder
 RUN apt-get update && apt-get install -y build-essential cmake libsparsehash-dev
-RUN mkdir -p /app/build
 COPY . /app/
 WORKDIR /app/build/
 RUN cmake -DCMAKE_BUILD_TYPE=Release .. && make -j 10
```

Comment:
Instead of using a fixed value for the parameter `-j` one could use `-j $(nproc)`.

Reply:
Good idea